deployments/keycloak-security/configs/proxy/traefik.toml

```diff
@@ -0,0 +1,32 @@
+defaultEntryPoints = ["http","https"]
+    logLevel = "INFO"
+    [entryPoints]
+      [entryPoints.http]
+        address = ":80"
+        compress = true
+
+      [entryPoints.https]
+        address = ":443"
+        compress = true
+        [entryPoints.https.tls]
+          [entryPoints.https.tls.defaultCertificate]
+          certFile = "/cert.pem"
+          keyFile = "/key.pem"
```

Comment:
❓ Is there any reason for exposing http if we allow https?

Reply:
Ya, the traefik's https isn't actually working at the moment. It's http only for now, I'll add this in properly in my next PR where I'm adding TLS to the gateway as well